tests/unit/test_sentry.py

```
@@ -0,0 +1,503 @@
+                    "username": "alice",
+                }
+            }
+        }
+        hint: dict[str, Any] = {}
+
+        result = before_send_hook(cast("Event", event), cast("Hint", hint))
+
+        assert result is not None
+        result_dict = cast("dict[str, Any]", result)
+        data = result_dict["request"]["data"]
+        assert data["password"] == "[REDACTED]"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "password"

(S105)